src/collections/blog/2023/2023-05-22-changing-release-channels-in-meshery/index.mdx

```diff
@@ -0,0 +1,177 @@
+  different release channels, so that improved controls may be provided to both
+  Meshery users and Meshery developers. The two release channels are edge and
+  stable release channels. Relative to stable releases, edge releases occur much
+  more frequently. Edge releases are made with each merge to master, unless that
+  merge to master is for a stable release. Stable releases are made with each
+  merge to master when a GitHub release tag is also present in the workflow.
+</p>
+
+<h3> How release channels offer subscription </h3>
+<p>
+  Release Channels offers a subsciption where user can subscribe to a specific 
+  release channel and get notified when a new release is available. This is  
+  useful for users who want to stay up to date with the latest Meshery features, while also
+  also providing flexibility for users who want to stay on a specific version of Meshery.
+
+  However, this approach can be risky because some updates may introduce bugs or 
+  compatibility issues that could break your existing installation. 
+  In such cases, having a subscription means that you will automatically 
+  receive these updates even if they are not suitable for your use case.
+
+  On the other hand, release channels also offer the ability to pin to a specific release which is a 
+  good thing as it allows users to maintain stability and predictability of their environment by preventing 
+  unexpected changes from being introduced into their system. However, doing so cancels out any future subscription-based 
+  benefits such as receiving security patches or bug fixes that were added after that version was released.
+
+  Therefore, it's important for users to weigh the pros and cons of each option before making decisions on how they 
+  want to manage their Meshery deployment. It's recommended that organizations have a well-defined upgrade 
+  strategy based on testing and validation procedures prior to applying new releases in production environments whether 
+  via subscriptions or manual upgrades to ensure 
+  that system availability is maintained and risks are minimized.
+</p>
+<p>
+  There are two ways to switch between Meshery release channels: using
+  mesheryctl or by editing your meshconfig file. In this blog post, we'll cover
+  both methods.
+</p>
+
+<br />
+<h3>
+  What is Meshconfig?
+</h3>
+<p>
+  Meshconfig is a configuration file that is used to configure Meshery. It is
+  typically located in the <code>~/.meshery</code> directory.It contains
+  information about the current release channel, the version of Meshery that is
+  installed, and other configuration options that are specific to your Meshery
+  installation. It is a YAML file, so you can edit it using any text editor.
+
+  Meshconfig is automatically generated when you run Meshery for the first time.
+  It is also automatically updated when you update Meshery. Meshconfig is a YAML
+  file, so you can edit it using any text editor. 
+</p>
```

Comment:
I would be best if we present use of `mesheryctl system` as the ideal approach to manipulating your meshconfig, leaving direct editing as a fallback solution should any given subcommand not meet the users' needs.

Reply:
As @leecalcote said, it's best to handle via command rather than hardcore manual work. This solution is preferred ONLY when necessary. We don't want to cause issues to the issues, do we? :)

Also, it's `~/.meshery/config.yaml`